suite-native/atoms/src/Modal/BottomModal.tsx

```diff
@@ -46,6 +63,60 @@ const modalHeaderStyle = prepareNativeStyle(utils => ({
     paddingVertical: utils.spacings.medium,
 }));
 
+const useAnimationStyles = ({
```

Comment:
Do you think it should still be in the component file? I would be up for moving this to some other location but I'll leave this up to you.

Reply:
Since modal is already a directory, I would create hooks.ts or animationHooks.ts and move it there.

Reply:
In my opinion this is getting pretty crowdy already 😕 You get too much overview when you only want to check something in the component. But that's just my view. If you think it's fine, just resolve this.